.config/webpack/base.js

```diff
@@ -19,12 +19,22 @@ module.exports = (preprocessor = {}, mode = 'development') => {
             path: path.resolve(__dirname, './../..', dashLibraryName),
             filename: '[name].js',
             library: dashLibraryName,
-            libraryTarget: 'window',
+            libraryTarget: 'umd'
```

Comment:
We don't want the dash table to be accessible through `const DataTable = window.dash_table.DataTable'` but rather through `import { DataTable } from 'dash-table'`